sequencer/src/api.rs

```diff
@@ -1247,7 +1245,11 @@ mod test {
         let states = std::array::from_fn(|_| state.clone());
 
         let mut network = TestNetwork::with_state(
-            Options::from(options::Http { port }).catchup(Default::default()),
+            Options::from(options::Http {
+                port,
+                max_connections: None,
+            })
```

Comment:
Nit: can make this less verbose and future proof with `Options::with_port(port)`